osu.Game/Screens/Tournament/Drawings.cs

```diff
@@ -47,11 +47,18 @@ public class Drawings : OsuScreen
 
         public ITeamList TeamList;
 
+        protected override DependencyContainer CreateLocalDependencies(DependencyContainer parent) => new DependencyContainer(parent);
+
         [BackgroundDependencyLoader]
-        private void load(TextureStore textures, Storage storage)
+        private void load(TextureStore textures, Storage storage, DependencyContainer dependencies)
         {
             this.storage = storage;
 
+            TextureStore flagStore = new TextureStore(new RawTextureLoaderStore(new NamespacedResourceStore<byte[]>(new StorageBackedResourceStore(storage), "Drawings")));

```

Comment:
i'd probably add the local store on a newline rather than use the ctor param for clarity.
also makes it very obvious as to the order of lookups.